hera_cal/utils.py

```diff
@@ -144,3 +145,60 @@ def get_aa_from_calfile(freqs, calfile, **kwargs):
 
     # generate aa
     return get_aa(freqs, **kwargs)
+
+
+def JD2LST(JD, longitude):
+    """
+    Input:
+    ------
+    JD : type=float, julian date of observation
+
+    longitude : type=float, longitude of observer in degrees East
+
+    Output:
+    -------
+    Local Apparent Sidreal Time [Hour Angle]
+    """
+    t = Time(JD, format='jd')
+    return t.sidereal_time('apparent', longitude=longitude).value
+
+
+def LST2JD(LST, start_JD, longitude):
+    """
+    calculate LST -> JD quickly via a linear fit
+
+    Input:
+    ------
+    LST : type=float, local apparent sidereal time [hour angle]
+
+    start_JD : type=int, integer JD to anchor LST2JD conversion
```

Comment:
Clarify what you mean by "anchor." I think you really want to say that the returned JD must always be later than `start_JD` but less than one day later.

Reply:
Or maybe call it the "julian day."